modules/installation-user-infra-machines-advanced-customizing-live-serial-console.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installation-user-infra-machines-advanced-customizing-live-{boot}-serial-console_{context}"]
+= Modifying a live install {boot-media} to enable the serial console
+
+On clusters installed with {product-title} 4.12 and above, the serial console is disabled by default and all output is written to the graphical console. You can enable the serial console with the following procedure.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[`coreos-installer` image mirror] page.
+
+ifeval::["{boot-media}" == "ISO image"]
+. Retrieve the {op-system} ISO image from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/[{op-system} image mirror] page and run the following command to customize the ISO image to enable the serial console to receive output:
++
+[source,terminal]
+----
+$ coreos-installer iso customize rhcos-<version>-live.x86_64.iso \
+  --dest-ignition <path> \ <1>
+  --dest-console tty0 \ <2>
+  --dest-console ttyS0,<baudrate> \ <3>
+  --dest-device /dev/sda <4>
+----
++
+<1> The location of the Ignition config to install.
+<2> The desired secondary console. In this case, the graphical console. Omitting this option will disable the graphical console.
+<3> The desired primary console. In this case, the serial console. If the baudrate is omitted, the default value of `9600` bps is used. For more information on the format of this option, see the link:https://www.kernel.org/doc/html/latest/admin-guide/serial-console.html[Linux kernel serial console] documentation.
```

Comment:
I wonder if we should suggest the default we use (`115200n8`) on most other platforms where we do encode a serial console. It's a saner default than the kernel's. Maybe something like:

```suggestion
<3> The desired primary console. In this case, the serial console. The `options` field defines the baud rate and other settings. A common value is `115200n8`. If no options are provided, the default kernel value of `9600n8` is used. For more information on the format of this option, see the link:https://www.kernel.org/doc/html/latest/admin-guide/serial-console.html[Linux kernel serial console] documentation.
```